styles/prosilver/template/mcp_cannedmessages_manage.html

```diff
@@ -1,6 +1,7 @@
 {% include 'mcp_header.html' %}
 {% INCLUDECSS '@phpbb_cannedmessages/cannedmessages.css' %}
-<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/editor.js -->
+{% INCLUDEJS T_ASSETS_PATH ~ '/javascript/editor.js' %}
```

Comment:
There's a js error coming up because of this include that I hadn't noticed before. Can we change this to:
```
{% if S_CANNEDMESSAGE_ADD_OR_EDIT %}
{% INCLUDEJS T_ASSETS_PATH ~ '/javascript/editor.js' %} 
{% endif %}
```